schema.go

```diff
@@ -340,6 +340,16 @@ var Schema = schema.NewTypedScopeSchema[*Config](
 				schema.PointerTo(util.JSONEncode(false)),
 				nil,
 			),
+			"ReadonlyRootfs": schema.NewPropertySchema(
+				schema.NewBoolSchema(),
+				schema.NewDisplayValue(schema.PointerTo("ReadonlyRootfs"), schema.PointerTo("Execute container process with or without a read only root file system"), nil),
+				false,
```

Comment:
Ah, yes. Good. We should have unit tests proving that the deployer module accepts true/false/missing ... I don't think the deployer tests have done all that much to prove that options passed in actually do what's intended (e.g., selinux labeling), but proving that we can/can't write to the root filesystem would be great.

Reply:
Proving that we can/can't write to the root filesystem sounds like a _functional_ test...but with a suitable mock, we _could_ have unit tests for this stuff which would be great!